addons/website/views/snippets/s_company_team.xml

```diff
@@ -7,8 +7,8 @@
             <div class="row s_nb_column_fixed">
                 <div class="col-lg-6 pt24 pb24">
                     <div class="row s_col_no_resize s_col_no_bgcolor">
-                        <div class="col-lg-4 pb16">
-                            <img alt="" src="/web/image/website.s_company_team_image_1" class="img-fluid rounded-circle mx-auto"/>
+                        <div class="col-lg-4 pb16 o_not_editable" contenteditable="false">
+                            <img alt="" src="/web/image/website.s_company_team_image_1" class="img-fluid rounded-circle mx-auto" contenteditable="true"/>
```

Comment:
This probably doesn't work after saving: the `contenteditable` attributes will be stripped and won't be added back in the next editing session. You probably need the `o_editable` class for that to work as intended, although you might need some additional work to make sure the editor handles nested `o_editable`/`o_not_editable` correctly.

Reply:
Not sure what is done here, but be careful: o_editable is not a class you should add by yourself. Only the o_not_editable can be added on elements you don't want to be editable and in that case contenteditable="false" would not be necessary... normally :grimacing:

Reply:
My understanding is that o_not_editable will only be processed on editor startup, is that not the case?

Reply:
Maybe, I am never sure. In that case, maybe contenteditable false on top of that is ok indeed